AUTHORS.md

```diff
@@ -65,12 +67,134 @@ CONTRIBUTORS
 
 Many others have participated and contributed to PCRE2 over its history.
 
-An incomplete list of recent contributors (alphabetical):
+The maintainers are grateful for all contributions and participation over the
+years. We apologise for any names we have forgotten.
 
-    Addison Crump
-    Alex Dowad
-    Carlo Marcelo Arenas Belón
+We are especially grateful to Philip Hazel, creator of PCRE and PCRE2, and
+maintainer from 1997 to 2024.
+
+All names listed alphabetically.
 
-You may browse Git's history for further details of many past contributors.
+### Contributors to PCRE2
 
-The maintainers are grateful for all contributions and participation over the years, whether or not listed here in this recent summary.
+This list is includes names up until the PCRE2 10.44 release. New names will be
```

Comment:
I thinl "is" is not gramatically correct here

Reply:
!! Well spotted, thanks.